index.html

```diff
@@ -2,60 +2,141 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <title>Fork Me! FCC: Test Suite Template</title>
+    <title>Apex Computer Systems</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+
     <link rel="stylesheet" href="css/style.css">
+ 
   </head>
 
   <body>
+    <header id="header">
+      <div id="main-logo">
+        <img id="header-img" src="images/apex-logo.svg" alt="logo"/>
+        <p>Apex Computer Systems</p>
+      </div>
+      <nav id="nav-bar">
+        <div>
+          <a href="#features" class="nav-link">Features</a>
+          <a href="#pricing" class="nav-link">Pricing</a>
+          <a href="#where-to-buy" class="nav-link">Where to Buy</a>
+        </div>
+      </nav>
+    </header>
+
+<div id="main-div">
+  
```

Comment:
kindly wrap content in  Html semantic tags ie. section tags, it makes your code more semantic